randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -177,17 +177,31 @@ Extra - asset_id: 4
   * Normal Door to The Moat/Door to West Ocean
   > Door to Wrecked Ship Entrance
       Any of the following:
-          # TODO: Check to see if possible with HBJ.
           Grapple Beam or Space Jump
-          # It is possible to jump across by not using full jumps to avoid bonking on the ceiling.
+          Horizontal Infinite Bomb Jump (Expert) and Can Lay Bombs
+          # It is possible to jump across by not using full jumps to avoid bonking on the celing.
           Movement (Intermediate)
           All of the following:
-              # From the sea floor
+              # From the Sea Floor
               Gravity Suit
               Any of the following:
                   Hi-Jump Boots or Wall Jumps (Beginner)
                   # IBJ
                   Vertical Infinite Bomb Jump (Beginner) and Can Lay Bombs
```

Comment:
Dumb idea, how difficult is IBJing from the water without gravity?

Reply:
You can't IBJ under water without gravity.